### PR TITLE
Implemented populating the table with time values, rest requests for …

### PR DESCRIPTION
**Pull request by ValdarRudman**
…day-time sign unsign and weekly. When supervisor has signes shows an iframe embedded with a preview image of the signed document. This was done over useing  as the image files will have to have unrestricted access to be viewed. Thi bonus with the iframe - allows user to open the preview in a new tab if so wishs.

### CODE REVIEW

File: src/app/tsweekly.service.ts [hunk 1]
```diff
@@ -57,17 +60,43 @@ export class TsweeklyService {
     }
 
     getWeeklies(email: string, weekid: number): Observable<TsWeekly> {
+
         console.log('Getting weeklies for', email, weekid);
-        return this.http.get<TsWeekly[]>(this.configUrl + '/' + email + '/' + weekid).pipe(
-            mergeMap((items: TsWeekly[]) => from(items)),
+
+        const token = 'Bearer ' + this.oAuthService.getIdToken();
```
Comment (SeanCondon) [inline src/app/tsweekly.service.ts:66]: I think you can use `localStorage.getItem` as you saved the token in there on login (`app.component.ts`) - same either way I guess

File: src/app/user-times/user-times.component.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+import {TsDay, TsdaysService} from '../tsdays.service';
+import {TsWeekly, TsweeklyService} from '../tsweekly.service';
+import {OAuthService} from 'angular-oauth2-oidc';
+import {generate} from 'rxjs';
+import {EMAIL_ATTR} from '../app.component';
+
+const msInDay = 24 * 60 * 60 * 1000;
+
+@Component({
+  selector: 'app-user-times',
```
Comment (SeanCondon) [inline src/app/user-times/user-times.component.ts:28]: I can't see where this is being used

File: src/index.html [hunk 3]
```diff
@@ -26,8 +26,10 @@
 <!--    <meta name="openidcissuer" content="$OPENIDCISSUER">-->
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.0/css/all.css"
           integrity="sha384-lZN37f5QGtY3VHgisS14W3ExzMWZxybE1SJSEsQp9S+oqd12jhcu+A56Ebc1zFSJ" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
-<body>
+<body class="mat-typography">
 <app-root [email]="'sean@opennetworking.org'"></app-root>
```
Comment (SeanCondon) [inline src/index.html:33]: probably should change this

**Review by SeanCondon — APPROVED** [review]
LGTM - I left a few minor comments

File: src/app/user-times/user-times.component.ts (hunk 2 above)
Comment (ValdarRudman) [reply src/app/user-times/user-times.component.ts:28]: Its used in the subscribe for tsweeksService

File: src/index.html (hunk 3 above)
Comment (ValdarRudman) [reply src/index.html:33]: I'll change this to the timesheets one

**Pull request merged**